task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: fbc-related-image-check
+spec:
+  params:
+    - name: IMAGE_URL
+      description: the fully qualified image name
+  results:
+    - name: HACBS_TEST_OUTPUT
+  workspaces:
+    - name: workspace
+  steps:
+    - name: check-related-images
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      workingDir: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      securityContext:
+        runAsUser: 1000
+      env:
+        - name: HOME
+          value: $(workspaces.workspace.path)/hacbs/$(context.task.name)
+      resources:
+        limits:
+          memory: 4Gi
+          cpu: 2
+        requests:
+          memory: 512Mi
+          cpu: 10m
+      script: |
+        #!/usr/bin/env bash
+        #Wanted to use extracted content from previous task, but for some ungodly reason you cannot create folder in workdir root directory
+        #unless its an emptydir volume wich is not transitioned to next step, because it is - emptydir
+        #I love tekton <3
```

Comment:
I don't think it has anything with tekton, IMO you are missing privileges.

May this will help, all tasks which are writing into workspace have this. Also some of them are running as root user
```
      securityContext:
        capabilities:
          add:
            - SETFCAP
```

Reply:
Tried both, SETFCAP and running as **root**, unfortunately made no difference at all.

Reply:
Needed to switch fbc-validation to run under root proviledge and removed emptydir volume. Now it works.